From 93cfd87f82362ae73b62602119b76d0bf7e38e01 Mon Sep 17 00:00:00 2001
From: He Zhe <zhe.he@windriver.com>
Date: Tue, 2 Feb 2016 02:04:06 -0500
Subject: [PATCH] Revert "drivers: net: Add Axxia NEMAC driver"

This reverts commit 15331fef6c5a51e99310708d46c59927d276b335.

"drivers: net: Add Axxia NEMAC driver" adds axxia/ to
drivers/net/ethernet/intel/Makefile, but there is no code under
the directory. This causes failure when kernel does a recursive
clean etc.

Revert CONFIG_AXXIA_NEMAC related patches, since there is no code
for it and no reference to it.

Signed-off-by: He Zhe <zhe.he@windriver.com>
Signed-off-by: Zhang Xiao <xiao.zhang@windriver.com>
---
 drivers/net/ethernet/intel/Kconfig  |   15 ---------------
 drivers/net/ethernet/intel/Makefile |    1 -
 2 files changed, 0 insertions(+), 16 deletions(-)

diff --git a/drivers/net/ethernet/intel/Kconfig b/drivers/net/ethernet/intel/Kconfig
index b10bc0c..f4ff465 100644
--- a/drivers/net/ethernet/intel/Kconfig
+++ b/drivers/net/ethernet/intel/Kconfig
@@ -17,21 +17,6 @@ config NET_VENDOR_INTEL
 
 if NET_VENDOR_INTEL
 
-config AXXIA_NEMAC
-	tristate "Axxia GigE NEMAC support"
-	default y
-	depends on ARCH_AXXIA && ARM64
-	select MII
-	---help---
-	  This driver supports the Axxia NEMAC gigabit Ethernet controller
-	  found on the Axxia AXM56xx family of devices.
-
-	  The NEMAC interface is a primarily used as a debug and management
-	  port.
-
-	  To compile this driver as a module, choose M here. The module
-	  will be called nemac.
-
 config E100
 	tristate "Intel(R) PRO/100+ support"
 	depends on PCI
diff --git a/drivers/net/ethernet/intel/Makefile b/drivers/net/ethernet/intel/Makefile
index f078175..5ea764d 100644
--- a/drivers/net/ethernet/intel/Makefile
+++ b/drivers/net/ethernet/intel/Makefile
@@ -2,7 +2,6 @@
 # Makefile for the Intel network device drivers.
 #
 
-obj-$(CONFIG_AXXIA_NEMAC) += axxia/
 obj-$(CONFIG_E100) += e100.o
 obj-$(CONFIG_E1000) += e1000/
 obj-$(CONFIG_E1000E) += e1000e/
-- 
1.7.5.4

